com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -169,7 +169,7 @@ internal set
         /// </summary>
         public bool DontDestroyWithOwner;
 
-        internal readonly HashSet<ulong> m_Observers = new HashSet<ulong>();
```

Comment:
`internal`s are treated as `public`

Reply:
> internals are treated as public
Is there an option/setting for it? Overall, the only thing that disturb me (just a very little) is the naming change of internal members.

Reply:
treating `internal` as `public` is by the standards but we can turn that check off if we really want.
also all the name changes are made by me manually, `dotnet-format` doesn't change your names.

quoting from the PR description:

> all changes in this PR are made according to errors indicated by `dotnet-format` applying `.editorconfig` rules.

and

> `dotnet-format` is able to format files and fix styling issues such as spaces, brackets etc.
> however, it is not capable of fixing name errors since they're strictly defined by the developer.